MAUI/DataGrid/selection.md

```diff
@@ -0,0 +1,547 @@
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        //Clear selection using selection mode
+        dataGrid.SelectionMode = Syncfusion.Maui.DataGrid.DataGridSelectionMode.None;
+
+        //Clear selection using clear selection method.
+        dataGrid.ClearSelection();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Events in selection
+The DataGrid provides the following events for selection:
+
+[SelectionChanging] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SelectionChanging): This event is raised while selecting a row at the execution time before the row is selected. So it allows canceling the selection action by setting the [Cancel] (https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?view=net-6.0#system-componentmodel-canceleventargs-cancel) property of [DataGridSelectionChangingEventArgs] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSelectionChangingEventArgs.html).
+[SelectionChanged] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SortColumnsChanged): This event is raised after a row is selected.
+These two events are triggered with `DataGridSelectionChangingEventArgs` and [DataGridSelectionChangedEventArgs] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSelectionChangedEventArgs.html) that contain the following properties:
+
+[AddedRows] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSelectionChangingEventArgs.html#Syncfusion_Maui_DataGrid_DataGridSelectionChangingEventArgs_AddedRows): Gets collection of the underlying data objects added for selection.
+[RemovedRows] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSelectionChangingEventArgs.html#Syncfusion_Maui_DataGrid_DataGridSelectionChangingEventArgs_RemovedRows): Gets collection of the underlying data objects removed from selection.
+
+To hook the SelectionChanging event and cancel the selection in following code example:
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        this.dataGrid.SelectionChanging += DataGrid_SelectionChanging;
+    }
+
+    private void DataGrid_SelectionChanging(object sender, Syncfusion.Maui.DataGrid.DataGridSelectionChangingEventArgs e)
+    {
+        e.Cancel = true;
+    }
+}
+{% enhighlight %}
+{% endtabs %}
+
+To get the selected item in code-behind using the `SelectionChanged` event, follow the code example:
```

Comment:
To get the selected item in a code-behind using the `SelectionChanged` event, follow this code example: